packages/ui-react/src/containers/AccountModal/forms/Registration.tsx

```diff
@@ -55,7 +55,9 @@ const Registration = () => {
     setConsentValues(extractConsentValues(publisherConsents));
   }, [accountController, publisherConsents]);
 
-  const socialLoginURLs = useSocialLoginUrls(window.location.href.split('?')[0]);
+  const socialLoginReturnURL = useMemo(() => `${window.location.href.split('/?')[0]}${modalURLFromLocation(location, 'personal-details')}`, [location]);
```

Comment:
I think it should also work:

const location = useLocation();

```
return modalURLFromLocation(location, 'personal-details')
```

Reply:
I don't want the rest of the query parameters to be kept, I want to make sure only `u=personal-details` is kept.

Reply:
Then it is still better to use `location.pathname` to avoid exposing window in the component.

Reply:
`window` was already there before me making this change, I assumed it's there for a reason. There are a couple of reasons I didn't want to get into re-evaluating its usage: `location.pathname` doesn't include the protocol prefix `https:` and that may be fine for relative navigation with React Router, but the API may need a full literal URL and I wouldn't risk breaking the functionality if it already works the way it is and keep the focus on what needs to be done incrementally. The other reason is that `modalURLFromLocation` doesn't remove the search params from `location`.
![image](https://github.com/jwplayer/ott-web-app/assets/46674160/29ea1647-9a09-4ed7-8c5b-791af492d16f)

Reply:
You can use `modalURLFromWindowLocation` to include the `https://`.
```javascript
modalURLFromWindowLocation('personal-details'); // https://app-preview.jwplayer.com/?app-config=2basfsic&u=personal-details
```

However, the query params are still kept, but also in your approach when appending `modalURLFromLocation`. Is that really an issue? I can imagine situations where we might still need them.